### PR TITLE
Add default tier to win dataplane

### PR DESCRIPTION
**Pull request by mazdakn**
## Description

Add support for tier's default action to windows dataplane.

## Related issues/PRs
Original PR: https://github.com/projectcalico/calico/pull/9085

## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
TBD
```

## Reminder for the reviewer

Make sure that this PR has the correct labels and milestone set.

Every PR needs one `docs-*` label.

- `docs-pr-required`: This change requires a change to the documentation that has not been completed yet.
- `docs-completed`: This change has all necessary documentation completed.
- `docs-not-required`: This change has no user-facing impact and requires no docs.

Every PR needs one `release-note-*` label.

- `release-note-required`: This PR has user-facing changes. Most PRs should have this label.
- `release-note-not-required`: This PR has no user-facing changes.

Other optional labels:

- `cherry-pick-candidate`: This PR should be cherry-picked to an earlier release. For bug fixes only.
- `needs-operator-pr`: This PR is related to install and requires a corresponding change to the operator.

### CODE REVIEW

File: felix/dataplane/windows/endpoint_mgr.go [hunk 1]
```diff
@@ -356,48 +357,54 @@ func (m *endpointManager) CompleteDeferredWork() error {
 			//
 			// Also note whether the default tier has policies or not. If the
 			// default tier does not have policies in a direction, then a profile should be added for that direction.
-			var ingressPolNames, egressPolNames [][]string
 			var defaultTierIngressAppliesToEP bool
 			var defaultTierEgressAppliesToEP bool
+			//var ingressRules, egressRules [][]*hns.ACLPolicy
```
[the remark this answers is not in the document]
Comment (mazdakn) [reply felix/dataplane/windows/endpoint_mgr.go]: Thanks. Removed.

File: felix/fv/winfv/policy_test.go [hunk 2]
```diff
@@ -178,13 +216,26 @@ var _ = Describe("Windows policy test", func() {
...
 
+			By("asserting destination is not reachable")
+			// Assert nginx-b is not reachable.
+			kubectlExecWithErrors(fmt.Sprintf(`-t porter -- powershell -Command 'Invoke-WebRequest -UseBasicParsing -TimeoutSec 5 %v'`, nginxB))
+
+			By("updating tier1 default action to pass")
+			tier1, err = client.Tiers().Get(context.Background(), tier1.Name, options.GetOptions{})
+			Expect(err).NotTo(HaveOccurred())
+			passAction := v3.Pass
+			tier1.Spec.DefaultAction = &passAction
```
Comment (song-jiang) [inline felix/fv/winfv/policy_test.go:234]: Suggest you add the same default pass rule to ingress policy tests as well.

Comment (mazdakn) [reply felix/fv/winfv/policy_test.go:234]: @song-jiang do you suggest to add a new test for ingress? Cause I don't see any other existing test.

Also, the tier default action works the same for both ingress and egress. And I tried to cover all combinations in the UTs.

Comment (song-jiang) [reply felix/fv/winfv/policy_test.go:234]: Agree egress policy test probably is good enough given we have UTs to cover all combinations.

**Pull request merged**